eng/templates/Azure.ServiceTemplate.Template/src/GlobalSuppressions.cs

```diff
@@ -0,0 +1,6 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("Usage", "AZC0002:DO ensure all service methods, both asynchronous and synchronous, take an optional CancellationToken parameter called cancellationToken.", Justification = "CancellationToken can be passed through RequestOptions")]
```

Comment:
Why do we need this suppression? I doubt someone will find and remove it, so the AZC0002 analyzer would be effectively disabled.

Reply:
In LLC, the rest method will use RequestContext which contains CancellationToken. But it will break AZC0002. In order to build successfully , need to add this suppression.